src/components/modal/Previewer.jsx

```
@@ -0,0 +1,39 @@
+import React from "react";
+import AppModal from "react-modal";
+import { useSelector } from "react-redux";
+
+const Previewer = ({ onClose, isOpen, children }) => {
```

ESLint

[error] 'onClose' is missing in props validation

(react/prop-types)